micro/iglu.json

```diff
@@ -0,0 +1,28 @@
+{
+	"schema": "iglu:com.snowplowanalytics.iglu/resolver-config/jsonschema/1-0-0",
+	"data": {
+		"cacheSize": 500,
+		"repositories": [
+			{
+				"name": "Iglu Central",
+				"priority": 0,
+				"vendorPrefixes": ["com.snowplowanalytics", "dev.amp.snowplow"],
+				"connection": {
+					"http": {
+						"uri": "http://iglucentral.com"
+					}
+				}
+			},
+			{
+				"name": "Washington City Paper - Dev",
+				"priority": 9,
```

Comment:
curious what this priority num means?

Reply:
Enrichment process reads through each object depending on priority. Lower number = higher priority. 

`Priority` Helps the schema resolver determine the lookup order for the schema repositories. See` [Iglu Schema Resolution](https://docs.snowplowanalytics.com/docs/pipeline-components-and-applications/iglu/common-architecture/schema-resolution/#3-registry-priority)